Node.js cannot find module

## Description

When I use jest to test, I get the error:

`Cannot find module '../build/Release' from '../node_modules/@iota/client/lib/index.js'`

## Fix
After I modify the following code in `index.js`, it works.
https://github.com/iotaledger/iota.rs/blob/b4e04e735b96c3345037a4106f5ab3877cdc9a30/bindings/nodejs/lib/index.js#L13
Change `require('../build/Release') `to `require('../build/Release/index.node')`

--- a/bindings/nodejs/lib/index.js
+++ b/bindings/nodejs/lib/index.js
@@ -10,7 +10,7 @@ const {
   UnspentAddressGetter,
   AddressGetter,
   BalanceGetter
-} = require('../build/Release')
+} = require('../build/Release/index.node')
 
 function promisify(fn, parse = true) {
   return function () {

